src/Compilers/CSharp/Test/Emit/PDB/CSharpPDBTestBase.cs

```diff
@@ -40,8 +40,12 @@ public static bool CheckIfSpanWithinSequencePoints(TextSpan span, string source,
             var endRow = endLine.LineNumber + 1;
             var endColumn = span.End - endLine.Start + 1;
 
-            var doc = new XmlDocument();
-            doc.LoadXml(pdb);
+            var doc = new XmlDocument() { XmlResolver = null };
+            using (var sr = new System.IO.StringReader(pdb))
+            using (var reader = new XmlTextReader(sr) { DtdProcessing = DtdProcessing.Prohibit })
```

Comment:
📝 `reader.Dispose()` already calls `sr.Dispose()`, so there is no need for nested `using` statements.

```suggestion
            using (var reader = new XmlTextReader(new StringReader(pdb)) { DtdProcessing = DtdProcessing.Prohibit })
```